pages/in/whitepaper.vue

```diff
@@ -87,18 +87,6 @@ const whitePaperSrc = [
     languageKey: 'cn',
     src: '/in/likecoin-whitepaper-cn.pdf',
   },
-  {
-    languageKey: 'ko',
-    src: '/in/likecoin-whitepaper-ko.pdf',
-  },
-  {
-    languageKey: 'ru',
-    src: '/in/likecoin-whitepaper-ru.pdf',
-  },
-  {
-    languageKey: 'vi',
-    src: '/in/likecoin-whitepaper-vi.pdf',
-  },
```

Comment:
Should remove cn & zh version as well?

Reply:
we happen to have the right version of cn and zh white paper in git

Reply:
but it seems u remove in this commit ??

![image](https://user-images.githubusercontent.com/17264716/85006899-54d56e00-b18d-11ea-98e5-2e1c98e6b30f.png)
![image](https://user-images.githubusercontent.com/17264716/85006917-5c951280-b18d-11ea-982c-91ffdfede3fc.png)

Reply:
No, those bytes are the diff, this file is much larger than 20 bytes

Reply:
The diffs are very small (Bytes instead of MB), indicating that it is only updating the file instead of removing it.
Not sure why Github shows `100%`